code/__HELPERS/jobs.dm

```diff
@@ -76,82 +76,82 @@
 
 // This returns a hud icon (from `hud.dmi`) by given job name.
 // Some custom title is from `PDApainter.dm`. You neec to check it if you're going to remove custom job.
-/proc/get_hud_by_jobname(jobname)
+/proc/get_hud_by_jobkey(jobname)
 	if(!jobname)
 		CRASH("The proc has taken a null value")
 
 	var/static/id_to_hud = list(
```

Comment:
huge lists like this should be on the job datum. That's why they exist. If you're going to refactor jobs, moving procs onto the job datum is a very good idea.

Reply:
It's not possible to remove this proc because it has customised HUD, but also Archanial suggested building that here instead of having var in each job.

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/87972842/35053cd6-d8d3-4ab1-9a7f-ddee439fb1c0)